### DISCUSSION TITLE
posts/why-are-you-not-improving/

### DISCUSSION THREAD

**Discussion by giscus** [MoldyMacaroniX/gd-website#53] [Comments]
# posts/why-are-you-not-improving/

It took me 5 years to beat my first extreme demon, here's what I should of done differently.

https://www.dashword.net/posts/why-are-you-not-improving/

**Comment by Domi-77** [comment]
I think these tips are quite helpful, but I think it is important to note to not push yourself too much, as it might seem kind of disappointing to not be able to do really difficult things
